### PR TITLE
ROMFS: add new VTOL config and mixer (generic quadplane tiltrotor VTOL)

### PR DESCRIPTION
**Pull request by sfuhrer**
**Describe problem solved by this pull request**
This adds a new VTOL config: Quadplane tiltrotor VTOL. It has 4 motors in quad X configuration, whereby all motors can be tilted 90° forward to also provide thrust in fixed-wing mode. As for aerodynamic surfaces, it has ailerons (separately controlled), an elevator and a rudder.

It's currently set up such that only the front motors are running during fixed-wing flight. If that's not desired then VT_FW_MOT_OFFID can be set to 0 (to not switch off any motors in fixed-wing). 

It should also be possible to use this config on an airframe with only 2 motors tilt-able, or with having multiple motors on the same tilt actuator, as the tilt is only used during the transition phases, and not during hover (as we do on other vehicles like the tricopter tiltrotor (e.g. Convergence)).

**Test data / coverage**
No tested yet. 

**Additional context**
In SITL we already have something similar (`make px4_sitl gazebo_tiltrotor`)

### CODE REVIEW

**Review by LorenzMeier — APPROVED** [review]

File: ROMFS/px4fmu_common/init.d/airframes/CMakeLists.txt [hunk 1]
```diff
@@ -137,6 +137,8 @@ px4_add_romfs_files(
 	13012_convergence
 	13013_deltaquad
 	13014_vtol_babyshark
+	13015_generic_vtol_standard
```
Comment (dagar) [inline ROMFS/px4fmu_common/init.d/airframes/CMakeLists.txt:140]:
```suggestion
```

**Comment by gulunehasret** [comment]
Does generic quadplane tiltrotor VTOL have only conservation tail configuration ? Does V or reverse V configurations are release?

**Comment by rollysing** [comment]
Hello,

Just wondering when this might become available.

Thanks.

**Comment by sfuhrer** [comment]
> Does generic quadplane tiltrotor VTOL have only conservation tail configuration ? Does V or reverse V configurations are release?

Yes, just for the "normal" tail configuration for now. You would need to adapt the mixer file slightly to make it work with a V tail.

**Comment by rollysing** [comment]
@sfuhrer Do we have an ETA for this to be available? I just tried stable v1.11.3 and QGC Dev 2021-05-06. At first it showed up but after the Apply and Restart and Powercycle, it has now disappeared.

Thank you in advance.

**Comment by sfuhrer** [comment]
> v1.11.3

v1.12 is now branched off and we have a first release candidate https://github.com/PX4/PX4-Autopilot/releases/tag/v1.12.0-rc1. If testing goes smooth then it should be released in some days (weeks?) I suppose

**Pull request merged**